fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -126,7 +126,7 @@
     /**
      * Default number of records to attempt to run in a single transaction.
      */
-    public static final int DEFAULT_LIMIT = 100;
+    public static final int DEFAULT_REC_NUM_LIMIT = 800;
```

Comment:
I believe, because this is a `public` constant, that this rename is an API-breaking change.

I think I like this name better, but I'm not sure it's worth changing the API for it.

Reply:
Besides, changing a default value sounds like a change that needs adopters' attention, probably it should also be treated as a breaking change.